components/Modal.tsx

```diff
@@ -1,7 +1,13 @@
 import { useRef, useEffect } from "react";
-import styles from "@/styles/components/modal.module.scss";
+import styles from "../styles/components/modal.module.scss";
 
-export default function Modal({ children, isOpen, setIsOpen }) {
+interface Props {
+	children?: React.ReactElement;
+	isOpen: any;
+	setIsOpen: any;
+  }
+
+const Modal : React.FC<Props>  = ({ children, isOpen, setIsOpen }:Props) => {
```

Comment:
same with earlier comment: Props repeating is redundant + component declaration consistency.
apply this to all instances

Reply:
noted, dont use React.FC anymore because of PropsWithChildren